source/PreferencesPanel.cpp

```diff
@@ -178,6 +195,33 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		if(zones[latest].Value().KeyName() != Command::MENU.KeyName())
 			Command::SetKey(zones[latest].Value(), 0);
 	}
+	else if(key == 'i' && page == 'p')
+	{
+		page = 'i';
+		if(!downloadedInfo)
+		{
+			PluginHelper::Download(
+				"https://raw.githubusercontent.com/endless-sky/endless-sky-plugins/master/generated/plugins.json",
```

Comment:
Can you make the URL a constant at the top of the file?

Reply:
done